src/Sulu/Bundle/AdminBundle/Resources/js/stores/MultiSelectionStore/MultiSelectionStore.js

```diff
@@ -51,7 +51,13 @@ export default class MultiSelectionStore<T = string | number, U: {id: T} = Objec
         this.setLoading(true);
         return ResourceRequester.getList(this.resourceKey, {
             locale: this.locale ? this.locale.get() : undefined,
-            [this.idFilterParameter]: itemIds.join(','),
+            [this.idFilterParameter]: itemIds.map((itemId) => {
+                if (typeof itemId !== 'string') {
+                    return itemId;
+                }
+
+                return itemId.replace(/[/,]/g, '/$&');
```

Comment:
Shouldn't the comma be escaped by a backslash?

Reply:
And what exactly are you replacing it with?

Reply:
It will be replaced by itself preceded by a slash

Reply:
I agree, that the escape character should be a backslash normally, but it's a lot more difficult to handle, because if the request contains two backslashes, php treats it as one. If the request contains two slashes, php also treats it as two slashes. This is a lot more predictable.